wordpress-mobile/WordPress-Android
# Issue/11142 show low resolution image on the preview screen

Fixes #11142

This PR loads a low resolution image into `ImageEditor` module's ImageView using main app's image loader. 

P.S. Low resolution image url is generated using Photon service and device's width. This url needs to be created explicitly as unlike iOS we don't yet have a common image loader between WPAndroid and ReactNative editor.

**Steps to test**
- Select a site having a post with an image and using block editor
- Select the post and click the image on the post
- Low res image should display on the next screen

Test for
- Image resolution and loading time (we might need to figure out optimal resize factor).
- Image rendering on configuration and app/ fragment lifecycle changes like device rotation, app moving from background to foreground, screen lock/ unlock.
- Image rendering on Photon capable site and other types of sites I might not be aware of.

PR submission checklist:
